actors/evm/src/interpreter/system.rs

```diff
@@ -111,44 +111,70 @@ pub struct System<'r, RT: Runtime> {
     pub readonly: bool,
     /// Randomness taken from the current epoch of chain randomness
     randomness: OnceCell<[u8; 32]>,
+
+    /// This is "some" if the actor is currently a "zombie". I.e., it has selfdestructed, but the
+    /// current message is still executing.
+    tombstone: Option<Tombstone>,
```

Comment:
For more context, I suggest to add a link to the issue where the it became evident that we needed to track and special-case in-progress deletion. Probably https://github.com/filecoin-project/ref-fvm/issues/1174?

Alternative we can just describe why here.